server/doi/__tests__/api.test.ts

```diff
@@ -109,6 +109,7 @@ it('lets community admins create a DOI for pubs in their community', async () =>
 	});
 
 	expect(dois.pub).toEqual(expectedPubDoi);
+	// @ts-expect-error FIXME: Property 'deposit' does not exist on type 'object'.
```

Comment:
`depositJson` is not typed (was `any` first, now `object`). I don't know what the type is supposed to be, so just `ts-expect-error`ed it for now. If there is a type for this somewhere I can update it!